cli/cache/parsed_source.rs

```diff
@@ -218,14 +218,38 @@ impl ParsedSourceCacheModuleAnalyzer {
     let mut stmt = self.conn.prepare_cached(sql)?;
     stmt.execute(params![
       specifier.as_str(),
-      &media_type.to_string(),
+      serialize_media_type(media_type),
       &source_hash,
       &serde_json::to_string(&module_info)?,
     ])?;
     Ok(())
   }
 }
 
+// todo(dsherret): change this to be stored as an integer next time
+// the cache version is bumped
```

Comment:
I was wanting to look into whether storing module_info as binary would be faster, then bump the cache after that. I probably won't look into that for a while though.